### ISSUE TITLE
Crash for unknown death (NullReference tied to Jetpacks)

### ISSUE THREAD

**Issue by CiocilteaRazvan** [EliteMasterEric/Coroner#1]
I got killed by the shop rocket landing on me and a Null Reference exception was thrown in the log. The game end screen didn't show up after the ship took off and we were stuck. Sorry, I closed the game and I don't have the logs. Perhaps a try/catch for situations like these where the cause of death was not coded yet.

**Comment by daakru** [comment]
this is possibly related, I was playing single player in Online mode on the first day of second quota, I died to a thumper and it hit a Null Reference exception and froze the game making me force close it, tried again with no changes to configs or mods and died from falling and got a similar error; will test more to determine if it is related to the mod but this is the error it had in the console window:
```
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Coroner.AdvancedDeathTracker.IsHoldingJetpack (GameNetcodeStuff.PlayerControllerB playerController) (at <155d55c0222d45168e98ad8b03967082>:0)
Coroner.AdvancedDeathTracker.GuessCauseOfDeath (GameNetcodeStuff.PlayerControllerB playerController) (at <155d55c0222d45168e98ad8b03967082>:0)
Coroner.AdvancedDeathTracker.GetCauseOfDeath (GameNetcodeStuff.PlayerControllerB playerController) (at <155d55c0222d45168e98ad8b03967082>:0)
Coroner.Patch.HUDManagerFillEndGameStatsPatch.Postfix (HUDManager __instance) (at <155d55c0222d45168e98ad8b03967082>:0) (wrapper dynamic-method) HUDManager.DMD<HUDManager::FillEndGameStats>(HUDManager,EndOfGameStats)
StartOfRound.EndOfGameClientRpc (System.Int32 bodiesInsured, System.Int32 daysPlayersSurvived, System.Int32 connectedPlayersOnServer) (at <39ddadb2aa5843e586a6c642ee60b0b5>:0)
StartOfRound+<unloadSceneForAllPlayers>d__239.MoveNext () (at <39ddadb2aa5843e586a6c642ee60b0b5>:0)
UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <e27997765c1848b09d8073e5d642717a>:0)
```

**Comment by daakru** [comment]
I tried changing the Coroner settings back to defaults, went back to Experimentation again, jumped off a rail and got another null reference exception:
![image](https://github.com/EliteMasterEric/Coroner/assets/22659210/972b596c-3a9f-435d-be14-6415a5bb8170)

**Comment by EliteMasterEric** [comment]
The mod is straight up broken right now I'm working on a patch

**Comment by EliteMasterEric** [comment]
This should be resolved in Coroner 1.1.0, which released a couple of days ago.
